rest/utilities_testing.go

```diff
@@ -232,7 +240,36 @@ func (rt *RestTester) Bucket() base.Bucket {
 
 		rt.DatabaseConfig.SGReplicateEnabled = base.BoolPtr(rt.RestTesterConfig.sgReplicateEnabled)
 
-		_, err = rt.RestTesterServerContext.AddDatabaseFromConfig(*rt.DatabaseConfig)
+		if rt.leakyBucketConfig != nil {
+			// Scopes and collections have to be set on the bucket being passed in for the db to use.
+			if rt.createScopesAndCollections {
```

Comment:
I'm not sure why this code is inside a createScopesAndCollections, unless 'createScopesAndCollections' is now equivalent to 'useScopesAndCollections' for the RestTester?